src/components/device/ha-device-automation-picker.ts

```diff
@@ -103,6 +104,7 @@ export abstract class HaDeviceAutomationPicker<
         .label=${this.label}
         .value=${value}
         @selected=${this._automationChanged}
+        @closed=${stopPropagation}
```

Comment:
An unrelated bug that causes the config flow form to close every time you select a device action in the action selector.